.github/semantic.yml

```diff
@@ -1,5 +1,5 @@
 # WARNING! THIS FILE IS AUTOGENERATED! DO NOT EDIT!
-# Orginal file: https://github.com/timberio/resources/tree/master/vector/github/semantic_pull_requests.tf
+# Original file: https://github.com/timberio/resources/tree/master/vector/github/semantic_pull_requests.tf
```

Comment:
The original file link here is dead.

Reply:
Ah, this is actually just a private repo - might be worth commenting that 👀

Reply:
Note - I opened a PR on the `resources` repo that corrects these as well, to avoid TF misspelling them again.